### PR TITLE
Exception fix

### PR DESCRIPTION
**Pull request by europaul**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,35 +6,35 @@ VERSION_NAME=v1.0.0
 lint:
 	black src/
 	flake8 src/
-.phony: lint
+.PHONY: lint
```
Comment (nithinsaii) [inline Makefile:9]: why all caps this?

File: Makefile [hunk 2]
```diff
@@ -6,35 +6,35 @@ VERSION_NAME=v1.0.0
...
 
 create_image:
 	docker build -t ${MODULE}:${VERSION_NAME} . -f docker/Dockerfile
-.phony: create_image
+.PHONY: create_image
 
 run_image:
-	docker run --device=/dev/ttyUSB0  --rm --env-file=./config.env ${MODULE}:${VERSION_NAME}
-.phony: run_image
+	docker run --device=/dev/ttyACM0 --rm --env-file=./.env ${MODULE}:${VERSION_NAME}
```
Comment (nithinsaii) [inline Makefile:20]: why this change?

File: Makefile (hunk 1 above)
Comment (europaul) [reply Makefile:13]: that's the syntax (https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html). otherwise it doesn't work

File: Makefile (hunk 2 above)
Comment (europaul) [reply Makefile:24]: `.env` because that's what we use in other modules. acm instead of usb because that is the name linux gives to the ST boards we are currently using

**Review by nithinsaii — APPROVED** [review]
looks good.
